pkg/database/metrics.go

```diff
@@ -0,0 +1,33 @@
+package database
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/util/recover"
+)
+
+func (db Database) emitMetrics(ctx context.Context) {
+	t := time.NewTicker(time.Minute)
+	defer t.Stop()
+
+	for {
+		i, err := db.OpenShiftClusters.QueueLength(ctx)
+		if err != nil {
+			db.log.Error(err)
+		}
+		defer recover.Panic(db.log)
+
+		db.m.EmitGauge("database.queue.openshiftclusters.length", int64(i), nil)
+
+		select {
+		case <-t.C:
+		case <-ctx.Done():
```

Comment:
for now, remove this, but open a follow-up issue to investigate changing our use of stop channels to contexts.

Reply:
https://github.com/Azure/ARO-RP/issues/96